tuning/trainercontroller/callback.py

```diff
@@ -59,6 +67,21 @@
 CONTROLLER_OPERATIONS_KEY = OPERATIONS_KEY
 
 
+def get_evaluator(metrics: dict) -> EvalWithCompoundTypes:
```

Comment:
Any thoughts on making this part of the class, e.g., as a staticmethod? It feels a bit strange to me to store it separately here like this

Reply:
Making it a static method would require overriding the class. Since this is just a utility function I will move it to a more appropriate location. We can also remove the function entirely if necessary.